pkg/api/v1/testkube/model_windows_security_context_options.go

```diff
@@ -0,0 +1,17 @@
+/*
+ * Testkube API
+ *
+ * Testkube provides a Kubernetes-native framework for test definition, execution and results
+ *
+ * API version: 1.0.0
+ * Contact: testkube@kubeshop.io
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package testkube
+
+type WindowsSecurityContextOptions struct {
```

Comment:
Not sure if it makes sense to support `WindowsOptions`, given that Testkube doesn't support Windows executions

Reply:
we have it in merge, in worse case, it will not be used